crates/frontend/src/utils.rs

```diff
@@ -424,3 +425,29 @@ pub fn unwrap_option_or_default_with_error<T>(option: Option<T>, default: T) ->
         default
     })
 }
+
+pub fn get_local_storage(_key: &str) -> Option<String> {
```

Comment:
@mahatoankitkumar , it would be better if we make this function generic, which will deserialize the value to a given type and then return it.

What do you think?

Reply:
@ShubhranshuSanjeev you can't store objects in localstorage, only strings. 
https://stackoverflow.com/questions/2010892/how-to-store-objects-in-html5-localstorage-sessionstorage

Reply:
lets leave it up to the calling function to decide the type after parsing it from strings

Reply:
@Datron @ShubhranshuSanjeev Lets decide here first that who will decide the type after parsing it from string (the calling function or the called function) 🥲

Reply:
> @ShubhranshuSanjeev you can't store objects in localstorage, only strings.

Yes @Datron , I am aware of that, we set the value to a string only, but serde_json can parse those string to a type.

Reply:
>@Datron @ShubhranshuSanjeev Lets decide here first that who will decide the type after parsing it from string (the calling function or the called function)

@mahatoankitkumar , didn't really understood the question, if the function is generic then the caller decides the return type, otherwise the called function will dictate the return type.

While fetching from localstorage, we will always get a string, and depending on the requirement we will have to parse value to a certain type, doing this inside get_local_storage will reduce the repetition.

Debate here is for sure whether we want couple the parsing to the `get`, and as I can see parsing is going to be a subsequent step of `get_local_storage`, as everything is a `String`.